extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -84,6 +88,7 @@ public function extendImageData( $data, $width ) {
 			'height' => $dimensions[ 'height' ],
 			'width' => $dimensions[ 'width' ],
 			'thumbnail' => $thumbnail->getUrl(),
+			'thumbnail2x' => $thumbnail2x->getUrl(),
 			'key' => urlencode( $data[ 'key' ] ),
 			'media-type' => $data[ 'isVideo' ] ? 'video' : 'image',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "image" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)